lib/utils.js

```diff
@@ -124,12 +124,12 @@ module.exports = {
         };
 
         return {
-            launch: options.launch || DEFAULT_LAUNCH_NAME,
-            description: options.description,
+            launch: options.launch || options.reportportalAgentJsPostmanLaunch || DEFAULT_LAUNCH_NAME,
+            description: options.description || options.reportportalAgentJsPostmanDescription,
             attributes: options.attributes ? options.attributes.concat(systemAttr) : [systemAttr],
```

Comment:
Don't we need to use `options.reportportalAgentJsPostmanAttributes` here?

Reply:
The attributes will be implemented as part of the other task [EPMRPP-61095](https://jiraeu.epam.com/browse/EPMRPP-61095)

Reply:
Got it.